lib/Doctrine/DBAL/Sharding/PoolingShardConnection.php

```diff
@@ -65,17 +66,17 @@
 class PoolingShardConnection extends Connection
 {
     /**
-     * @var array
+     * @var DriverConnection[]|null
```

Comment:
The variable could be initialized with an empty array instead.

Reply:
Probably ok here and for `$connections`, changed (`close()` now sets empty array instead of null).